Bottom: 4e27b8682d98180fa02f09c9e9fc3f9138ab980d
Top:    a6f7969dbc150bf251f89237248f5aa7bd1fd70f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 17:12:27 +0000

hid/lpr: Don't mis-match popen and fclose (should be pclose)


---

diff --git a/src/hid/lpr/lpr.c b/src/hid/lpr/lpr.c
index 259812c..fc9403b 100644
--- a/src/hid/lpr/lpr.c
+++ b/src/hid/lpr/lpr.c
@@ -106,7 +106,7 @@ lpr_do_export (HID_Attr_Val * options)
 
   ps_hid_export_to_file (f, options);
 
-  fclose (f);
+  pclose (f);
 }
 
 static void
